### PR TITLE
Add spider for Osasco/SP

### PR DESCRIPTION
**Pull request by alfakini**
Hey people,

In this PR we add the spider to collect data from [Osasco/SP gazette](http://www.osasco.sp.gov.br/imprensa-oficial/).

### CODE REVIEW

**Comment by alfakini** [comment]
Tests are failing on setup:

```sh
0.01s$ sudo rm /usr/local/bin/docker-compose
406.56s$ curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
curl: (56) SSL read: error:00000000:lib(0):func(0):reason(0), errno 104
The command "curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose" failed and exited with 56 during .
Your build has been stopped.
```

File: processing/data_collection/gazette/spiders/sp_osasco.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    def parse(self, response):
+        """
+        @url http://www.osasco.sp.gov.br/imprensa-oficial
+        @returns requests 1
+        @scrapes date file_urls is_extra_edition territory_id power scraped_at
+        """
+
+        for element in response.css(self.GAZETTE_CSS):
+            url = element.css(self.URL_CSS).extract_first()
+            date = element.css(self.DATE_CSS).extract_first()
```
Comment (giovanisleite) [inline processing/data_collection/gazette/spiders/sp_osasco.py]: The upload date isn't always the document date. [Here](http://www.osasco.sp.gov.br/imprensa-oficial?page=12) you can see two files uploaded with the same date, but the documents date isn't the same.

**Comment by jaswdr** [comment]
@alfakini any plans to continue this PR?

**Comment by engFelipeMonteiro** [comment]
there is a fix, for this PR,  #522 .
this one should be closed

**Comment by rennerocha** [comment]
Closing stale PR.

**Pull request closed without merging**